FDK/Tools/SharedData/FDKScripts/CheckOutlinesUFO.py

```diff
@@ -1,1031 +1,1164 @@
-__copyright__ = """Copyright 2015, 2016 Adobe Systems Incorporated (http://www.adobe.com/). All Rights Reserved.
-"""
+from __future__ import print_function
 
-__usage__ = """
-   checkOutlinesUFO program v1.21 Aug 8 2017
+import hashlib
+import os
+import re
+import shutil
+import subprocess
+import sys
+
+import booleanOperations.booleanGlyph
+import defcon
+import ufoLib
+from robofab.pens.digestPen import DigestPointPen
```

Comment:
(for a later round) the replacement for this is at https://github.com/robofab-developers/fontPens